src/tbp/monty/frameworks/models/evidence_matching/hypotheses_displacer.py

```diff
@@ -381,7 +381,7 @@ def _get_pose_evidence_matrix(
             # nodes where pc1==pc2 receive no cd evidence but twice the pn evidence
             # -> overall evidence can be in range [-1, 1]
             cd1_evidence = cd1_evidence * use_cd
-            pn_evidence[np.logical_not(use_cd)] * 2
+            pn_evidence[np.logical_not(use_cd)] *= 2
```

Comment:
unasked for opinion: as this is a breaking change anyway, maybe we should rename this to `surface_normal_evidence`